bindings/nodejs/lib/types/block/input/input.ts

```diff
@@ -35,7 +35,7 @@ class TreasuryInput extends Input {
     /**
      * The milestone id of the input.
      */
-    milestoneId: HexEncodedString;
```

Comment:
This is a breaking change, public to readonly, no?

Reply:
Hmm good catch, there are actually more changes like that in the PR

Reply:
Yes, all of them should stay public

Reply:
@abdulmth Can you open an issue where we'd list all breaking changes we'd do for a 2.0 version please? And add this